records_mover/records/validated_records_hints.py

```diff
@@ -110,13 +111,25 @@ def validate_timeonlyformat() -> HintTimeOnlyFormat:
                                     'timeonlyformat',
                                     default="HH24:MI:SS")
 
+        T = TypeVar('T')
+
+        def validate_literal_type(type_: Type[T],
+                                  default: T,
+                                  hint_name: str) -> T:
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker